projects/plugins/jetpack/extensions/blocks/recurring-payments/test/fixtures/jetpack__recurring-payments.html

```diff
@@ -1,3 +1,9 @@
-<!-- wp:jetpack/recurring-payments {"planId":12,"align":"right","url":"?open_payment_plan=12","uniqueId":"recurring-payments-12"} -->
-<div class="wp-block-jetpack-recurring-payments"><!-- wp:jetpack/button {"element":"a","uniqueId":"recurring-payments-12","text":"Payment","url":"?open_payment_plan=12","borderRadius":0} /--></div>
-<!-- /wp:jetpack/recurring-payments -->
\ No newline at end of file
+<!-- wp:jetpack/recurring-payments {"layout":{"type":"flex","justifyContent":"center"},"style":{"spacing":{"blockGap":"20px"}}} -->
+<div class="wp-block-jetpack-recurring-payments"><!-- wp:jetpack/recurring-payments-button {"planId":762} -->
```

Comment:
Why do we have a plan Id associated to the button container?

Reply:
So maybe there's a confusion over the structure here.

jetpack/recurring-payments - container for multiple jetpack/recurring-payment.
jetpack/recurring-payments-button - associates individual buttons with plans 
jetpack/button - implements actual, general purpose buttons